plexupdate.sh

```diff
@@ -307,6 +307,10 @@ if [ -z "${DISTRO_INSTALL}" ]; then
 			else
 				DISTRO_INSTALL="${REDHAT_INSTALL}"
 			fi
+		elif [ -f /etc/synoinfo.conf ]; then
+			DISTRO="synology"
+			BUILD="linux-${ARCH}"
```

Comment:
`linux-${ARCH}` is already set as the default in line 56, so this line can be removed.
```suggestion
```